Sming/Core/DateTime.cpp

```diff
@@ -18,7 +18,7 @@
 static_assert(sizeof(time_t) != 8, "Great! Now supports 64-bit - please update code");
 #warning "**Y2038** time_t is only 32-bits in this build configuration"
 #else
```

Comment:
If GCC v10 is ok then wouldn't it be better to change this line to

```
#elif defined(__GNUC__) && __GNUG__ < 10
```

Reply:
Perhaps... but as this is technically a glibc issue the more correct way would be to check `__USE_TIME_BITS64` - see https://lwn.net/Articles/664800/.